include/zephyr/drivers/espi_emul.h

```diff
@@ -37,39 +37,39 @@ struct espi_emul;
  * Passes eSPI virtual wires set request (virtual wire packet) to the emulator.
  * The emulator updates the state (level) of its virtual wire.
  *
- * @param emul Emulator instance
+ * @param emulator The device Emulator instance
  * @param vw The signal to be set.
  * @param level The level of signal requested LOW(0) or HIGH(1).
  *
  * @retval 0 If successful.
  * @retval -EIO General input / output error.
  */
-typedef int (*emul_espi_api_set_vw)(struct espi_emul *emul, enum espi_vwire_signal vw,
+typedef int (*emul_espi_api_set_vw)(const struct emul *emulator, enum espi_vwire_signal vw,
```

Comment:
Use emu for the argument name to avoid making this so long

Reply:
Resolved in https://github.com/zephyrproject-rtos/zephyr/pull/46818

Had to create a new pull-request because GitHub won't let me update this pull-request after recreating my fork. Sorry.